.swiftlint.yml

```diff
@@ -2,7 +2,36 @@ disabled_rules:
  - type_name
 
 opt_in_rules:
+ - array_init
+ - attributes
+ - closure_end_indentation
+ - closure_spacing
+ - contains_over_first_not_nil
+ - discouraged_optional_boolean
+ - empty_count
+ - fatal_error_message
+ - first_where
  - force_unwrapping
+ - implicit_return
+ - implicitly_unwrapped_optional
+ - joined_default_parameter
+ - literal_expression_end_indentation
+ - multiline_arguments
+ - multiline_parameters
+ - operator_usage_whitespace
+ - overridden_super_call
+ - override_in_extension
+ - pattern_matching_keywords
```

Comment:
> Combine multiple pattern matching bindings by moving keywords out of tuples.

I'm not fan of this one. I actually personally prefer `case (let x, let y)` over `case let (x, y)` generally. Because if there's a variable named `y` before that line, the intent of `case let (x, y)` is unclear whereas if `x` is binded or just has to match the existing, previously-defined `y`…

Reply:
Good point, I'll remove it.